packages/docs/pages/select.tsx

```diff
@@ -1,399 +1,499 @@
+
+                      return (
+                        <Form>
+                          <FormGroup>
+                            <Select
+                              action={{
+                                actionType: 'destructive',
+                                content: 'Remove',
+                                description: 'Description for remove action',
+                                icon: <DeleteIcon />,
+                                onActionClick: () => null,
+                              }}
+                              label="Select"
+                              onOptionChange={() => null}
+                              options={[
+                                { value: 1, content: 'Option #1', description: 'Description for option #1' },
+                                {
+                                  value: 2,
+                                  content: 'Option #2',
+                                  description: 'Description for option #2',
+                                  disabled: true,
+                                },
+                                { value: 3, content: 'Option #3' },
+                                { value: 4, content: 'Option #4' },
+                                { value: 5, content: 'Option #5' },
+                              ]}
+                              onChange={handleChange}
+                              placeholder="Larger"
+                              required
+                              value={value}
+                            />
+                          </FormGroup>
+                        </Form>
+                      );
+                    }}
+                    {/* jsx-to-string:end */}
+                  </CodePreview>
+                </Fragment>
+              ),
+            },
+          ]}
+        />
+      </Panel>
+
+      <Panel header="Props" headerId="props">
+        <ContentRoutingTabs
+          id="props"
+          routes={[
+            {
+              id: 'select',
+              title: 'Select',
+              render: () => <SelectPropTable renderPanel={false} />,
+            },
+            {
+              id: 'select-option',
+              title: 'SelectOption',
+              render: () => <SelectOptionPropTable renderPanel={false} />,
+            },
+            {
+              id: 'select-action',
+              title: 'SelectAction',
+              render: () => <SelectActionPropTable renderPanel={false} />,
+            },
+            {
+              id: 'select-group',
+              title: 'SelectGroup',
+              render: () => <SelectGroupPropTable renderPanel={false} />,
+            },
+          ]}
+        />
+      </Panel>
 
-      <PageNavigation items={items} />
+      <Panel header="Do's and Don'ts" headerId="guidelines">
+        <GuidelinesTable
+          recommended={[
+            <>
+              Typically used within <NextLink href="/panel">Panels</NextLink>.
```

Comment:
```suggestion
              Typically use within <NextLink href="/panel">Panels</NextLink>.
```

Reply:
The suggestion is now phrase _wierd_. Could we potentially just drop `Typically`?

Reply:
FYI, I dropped the `Typically` 👍🏼